[ui] Create view to show an individual's info

evamillan opened the pull request:
This PR adds a view at `/individual/uuid` to show an individual's profile, identities and enrollments. A link to this view can be found in each individual's entry on the table, both in the buttons next to the name and in the dropdown menu.

![imagen](https://user-images.githubusercontent.com/26812577/181569805-c042dbbd-db7c-4bdf-84dd-512714f6020e.png)

![imagen](https://user-images.githubusercontent.com/26812577/181568997-5f9d054a-645e-41eb-a8b8-acb26b485489.png)

The list of identities in the expanded individual and the modal to enroll in an organization in the table and workspace are moved to new components so that they can be also used in this view. It also adds a button on the enrollment list to add one without using drag and drop.

Closes #628.

sduenas reviewed (changes requested):
The PR looks ok. However, I found a problem with the navigation. When I move to an individual view the history doesn't seem to be updated on the browser, so I can't go back to the main page.
Can you have it a look?

sduenas reviewed (changes requested):
We have decided that the icon for editing the name and email on the individual list should be removed because the full editing of the profile is going to be on the individual view.

Also, the icon "View full profile" should be also removed because the behavior will be to open the individual view when clicking on the individual's name.

sduenas reviewed (changes requested):
The view should also show the main UUID (i.e. mk field) of the individual.

evamillan commented:
I added the UUID to the view, removed the editable name and email fields from the table and replaced the icon with a link in the individual's name. We couldn't replicate the problem with the navigation, so nothing was changed in this regard.

evamillan commented:
I updated the PR to fix a bug that made the organizations data disappear when the individual was enrolled in a team.

jjmerchante commented:
LGTM Thank you!

zhquan reviewed (approved):
LGTM
